gazebo_ros_control:
  pid_gains:
    joint_1:
      p: 10.0
      i: 0.0
      d: 0.0
    joint_2:
      p: 10.0
      i: 0.0
      d: 0.0
    joint_3:
      p: 0.1
      i: 0.0
      d: 0.0
    joint_4:
      p: 0.1
      i: 0.0
      d: 0.0

# Controller configs (if any)
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 500

joint_1_controller:
  type: position_controllers/JointPositionController
  joint: joint_1

joint_2_controller:
  type: position_controllers/JointPositionController
  joint: joint_2

joint_3_controller:
  type: position_controllers/JointPositionController
  joint: joint_3

joint_4_controller:
  type: position_controllers/JointPositionController
  joint: joint_4
